src/Raven.Server/Web/Operations/GetSuggestConflictResolutionOperation.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Http;
+using Raven.Server.Json;
+using Raven.Server.Utils;
+using Sparrow.Json;
+
+namespace Raven.Server.Web.Operations
+{
+    internal class GetSuggestConflictResolutionOperation : IMaintenanceOperation<ConflictResolverAdvisor.MergeResult>
+    {
+        private readonly string _documentId;
+
+        public GetSuggestConflictResolutionOperation(string documentId)
+        {
+            _documentId = documentId;
+        }
+
+
+        public RavenCommand<ConflictResolverAdvisor.MergeResult> GetCommand(DocumentConventions conventions, JsonOperationContext context)
+        {
+            return new GetSuggestConflictResolutionCommand(_documentId);
+        }
+
+        public class GetSuggestConflictResolutionCommand : RavenCommand<ConflictResolverAdvisor.MergeResult>
```

Comment:
can be made private?

Reply:
@ppekrol No, I need it for `ShardedGetReplicationConflictsOperation` - but I can move the entire class (`GetSuggestConflictResolutionOperation`) to `ShardedReplicationHandlerProcessorForGetConflicts.cs`. Thoughts? :-)

Reply:
no need